### PR TITLE
Feature Input

### PR DESCRIPTION
**Pull request by Avin008**
### Input component Includes:

- Default Input

default input

- Input with Validation message

input error validation

input with validation messgae

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Everything is looking good and the code is well structured l.

File: Components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.input-group > label {
+  font-size: 0.8rem;
+  font-weight: 500;
+}
+
+.form-group > input:placeholder-shown {
+  font-size: 1rem;
+}
```
Comment (fahd23) [inline Components/input/input.css:36]: you can also have a class name for input and label right?

**Review by fahd23 — APPROVED** [review]
Look good to me

File: Components/input/input.css (hunk 1 above)
Comment (Avin008) [reply Components/input/input.css:36]: right! fixing it right away. Thanks for pointing it out.

**Pull request merged**